script.js

```diff
@@ -140,4 +140,47 @@ document.addEventListener('DOMContentLoaded', () => {
       event.run();
     }
   });
+  // Local storage for the form
+  const firstName = document.getElementById('name');
+  const emailAdd = document.getElementById('email');
+  const sms = document.getElementById('msg');
+
+  const storageForm = {
+    fName: '',
+    fEmail: '',
+    fMsg: '',
+  };
+
+  function populateStorage() {
+    storageForm.fName = firstName.value;
+    storageForm.fEmail = emailAdd.value;
+    storageForm.fMsg = sms.value;
+    localStorage.setItem('data', JSON.stringify(storageForm));
+  }
+
+  function getFormStorage() {
+    const reFill = JSON.parse(localStorage.getItem('data'));
+    firstName.value = reFill.fName;
```

Comment:
### Your Form and Error
![error](https://user-images.githubusercontent.com/93039359/181748378-d26aa25c-1bbe-42c1-b7f1-362d5eb91b9a.png)

- [ ] Your form is not setting and getting data from localStorage. Please fix this issue :+1:\